v2/tools/generator/internal/astmodel/package_reference.go

```diff
@@ -28,22 +28,11 @@ type PackageReference interface {
 	// IsPreview returns true if this package reference has a suffix indicating it's a preview
 	// release, false otherwise
 	IsPreview() bool
```

Comment:
This is also (at least with our current implementation) specific to internal refs, but possibly makes sense to leave here as it could in theory apply to external refs one day too...